Integrations/src/main/java/io/deephaven/integrations/learn/Computer.java

```diff
@@ -0,0 +1,86 @@
+package io.deephaven.integrations.learn;
+
+import io.deephaven.base.verify.Require;
+import io.deephaven.db.tables.Table;
+import io.deephaven.db.v2.sources.ColumnSource;
+import org.jpy.PyObject;
+
+/**
+ * Computer adds new updated/modified row indices to the index set, to be used in the deferred calculation.
+ */
+public class Computer {
+
+    private final PyObject modelFunc;
```

Comment:
Can avoid allocations and calculations if you store this as a python function thing.